### PR TITLE
Fixed latest news image on homepage

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**
Fixed latest news image on homepage.
Link: https://github.com/ita-social-projects/GreenCity/issues/818

### CODE REVIEW

File: src/app/component/home/components/eco-events/eco-events.component.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div id="eco-events-wrapper" *ngIf="latestNews.length > 0">
```
Comment (mplesha) [inline src/app/component/home/components/eco-events/eco-events.component.html:2]: What is the reason to use 'id' in your elements?

**Comment by mplesha** [comment]
```js
import { Component, OnInit, OnDestroy } from '@angular/core';
import { NewsService } from 'src/app/service/news/news.service';
import { NewsDto } from 'src/app/component/home/models/NewsDto';
import { of } from 'rxjs';
import { LanguageService } from 'src/app/i18n/language.service';
import { catchError } from 'rxjs/operators';
```
You now, what it means

**Comment by mplesha** [comment]
```html
<div id="main-event">
      <div id="main-event-image">
        <img [src]="latestNews[0]?.imagePath" alt="event-image">
      </div>
      <div id="main-event-content">
        <p class="event-date">{{ latestNews[0]?.creationDate }}</p>
        <h3>{{ latestNews[0]?.title }}</h3>
        <a class="read-more"
           [routerLink]="['/news', latestNews[2].id]"
        >
          {{ "homepage.eco-news.read-more" | translate }}
          <img [src]="arrow"
               alt="arrow"
          >
        </a>
      </div>
    </div>
    <div id="other-events" >
      <div class="other-event">
        <p class="event-date">{{ latestNews[1]?.creationDate }}</p>
        <h3>{{ latestNews[1]?.title }}</h3>
        <a class="read-more"
           [routerLink]="['/news', latestNews[2].id]"
        >
          {{ "homepage.eco-news.read-more" | translate }}
          <img [src]="arrow"
               alt="arrow"
          >
        </a>
      </div>
      <div class="splitter"></div>
      <div class="other-event">
        <p class="event-date">{{ latestNews[2]?.creationDate }}</p>
        <h3>{{ latestNews[2]?.title }}</h3>
        <a class="read-more"
           [routerLink]="['/news', latestNews[2].id]"
        >
          {{ "homepage.eco-news.read-more" | translate }}
          <img [src]="arrow"
               alt="arrow"
          >
        </a>
      </div>
    </div>
  </div>
  <a class="read-more centered"
     [routerLink]="['/news']"
  >
    {{ "homepage.eco-news.read-all" | translate }}
    <img [src]="arrow"
         alt="arrow"
    >
  </a>
```
Doubled, triplet code - actually, you have the same blocks which differ only in index

**Comment by mplesha** [comment]
```js
dateObj.getDate() +
      ' ' +
      localizedMonth.charAt(0).toUpperCase() +
      localizedMonth.slice(1, localizedMonth.length) +
      ' ' +
      dateObj.getFullYear()
```
Use es6 for concatenation strings

File: src/app/component/home/components/eco-events/eco-events.component.html (hunk 1 above)
Comment (RomanDeveloperAcc) [reply src/app/component/home/components/eco-events/eco-events.component.html]: > What is the reason to use 'id' in your elements?

Fixed

**Comment by RomanDeveloperAcc** [comment]
> ```js
> import { Component, OnInit, OnDestroy } from '@angular/core';
> import { NewsService } from 'src/app/service/news/news.service';
> import { NewsDto } from 'src/app/component/home/models/NewsDto';
> import { of } from 'rxjs';
> import { LanguageService } from 'src/app/i18n/language.service';
> import { catchError } from 'rxjs/operators';
> ```
> 
> You now, what it means

Yes, fixed

**Comment by RomanDeveloperAcc** [comment]
> ```js
> dateObj.getDate() +
>       ' ' +
>       localizedMonth.charAt(0).toUpperCase() +
>       localizedMonth.slice(1, localizedMonth.length) +
>       ' ' +
>       dateObj.getFullYear()
> ```
> 
> Use es6 for concatenation strings
Fixed

**Comment by mplesha** [comment]
Refactoring of the component  [eco-events](https://github.com/ita-social-projects/GreenCityClient/tree/dev/src/app/component/home/components/eco-events) (division into 2 components to avoid doubled code) will moved into other issue

**Pull request merged**